src/components/ebay-toggle-button-group/toggle-button-group.stories.js

```diff
@@ -0,0 +1,125 @@
+import { tagToString } from "../../../.storybook/storybook-code-source";
+import { addRenderBodies } from "../../../.storybook/utils";
+import readme from "./README.md";
+import component from "./index.marko";
+import withIconsTemplate from "./examples/icons.marko";
+import withIconsCode from "./examples/icons.marko?raw";
+import withDefaultTemplate from "./examples/withDefault.marko";
+import withDefaultCode from "./examples/withDefault.marko?raw";
+import controlledTemplate from "./examples/controlled.marko";
+import controlledCode from "./examples/controlled.marko?raw";
+
+const Template = (args) => ({
+    input: addRenderBodies(args),
+});
+
+export default {
+    title: "buttons/ebay-toggle-button-group",
+    component,
+    parameters: {
+        docs: {
+            description: {
+                component: readme,
+            },
+        },
+    },
+    argTypes: {
+        variant: {
+            type: "string",
+            control: { type: "select" },
+            options: ["checkbox", "radio", "single"],
```

Comment:
@LuLaValva , I agree that's not a great name; too oversimplified. Are there any concerns over user confusion about these variant names as a whole? Is it not better to avoid confusion with native behavior and avoid "checkbox" and "radio?" Why not keep it simple and aligned with Skin - 

1. Multi-select
2. Single-select optional
3. Single-select required

?

I think these options provide the most clarity directly instead of going through detours of other patterns for analogous behaviors that users will need to make connections with.

Reply:
After discussing offline, we decided to go with `checkbox`, `radio`, and `radio-toggle`